test/vstest.console.UnitTests/CommandLine/GenerateFakesUtilitiesTests.cs

```diff
@@ -37,7 +40,7 @@ public void FakesShouldNotBeGeneratedIfDisableAutoFakesSetToTrue()
         CommandLineOptions.Instance.DisableAutoFakes = true;
```

Comment:
Could we add a test for when disable is false?

Reply:
@vritant24 could you? :)

Reply:
Yup. I'm on it :)

Reply:
Should I just push it to this branch?

Reply:
Ah I now remember why we can't have that test. The code under test dynamically loads an assembly through reflection that isn't present in the test env. So testing a scenario where things should work won't be possible unless:
1. We introduce a nuget package that ships the assembly needed and reference it with the test project
2. We fake the static method `FakesUtilities.GenerateFakesSettingsForRunConfiguration` to not call into its logic and instead do something else.
3. We let it be.
Thoughts?